client/src/components/applicants/Applicants.js

```diff
@@ -35,10 +36,10 @@ export class Applicants extends Component {
     return (
       <div className='applicants'>
         <div className='applicants__title'>{jobPosting.title}</div>
-        <button className='applicants__button'>
-          Copy as a template
-        </button>
-        <div className='applicantion-stages'>
+        <div className='applicants__button' onClick={() => this.props.copyStages(stages)}>
```

Comment:
Tässä on nyt vähän hassusti sisäkkäin div, jolla on onClick toiminnallisuus ja sitten Button tuolta CopyStagesButton luokasta? Saavutettavuutta ajatellen (erityisesti ruudunlukuohjelmat) tässä olisi hyvä käyttää vaan buttonia divin sijaan. Ja siis ei MaterialUI buttonia. Sitten voisi varmaankin ottaa myös tuon toisen luokan buttonin pois kokonaan.